src/state/communication/messages/presentSlide.test.ts

```diff
@@ -57,10 +57,11 @@ describe('isValidPresentSlideMessage', () => {
     { view: null },
     { view: 111 },
     { view: {} },
-    { view: { slideId: undefined } },
-    { view: { slideId: null } },
-    { view: { slideId: 111 } },
-    { view: { slideId: '' } },
+    { view: { isEditMode: false, slideId: undefined } },
+    { view: { isEditMode: false, slideId: null } },
+    { view: { isEditMode: false, slideId: 111 } },
+    { view: { isEditMode: false, slideId: '' } },
+    { view: { isEditMode: undefined, slideId: 'slide-0' } },
```

Comment:
Please also check if `null` or a non-boolean like a number is rejected.

It seems tedious to test all these cases, but we found a lot of bugs in the validation schemas with these kinds of tests.